Makefile

```diff
@@ -64,6 +64,30 @@ install:
 		helm upgrade $(HELM_OPTS) $(CHART_INSTANCE) $(CHART_DIR); \
 	fi;
 
+.PHONY: install_giraffe
+install_giraffe: ## Installs the helm chart on the OpenShift cluster
+install_giraffe: check_environment
+install_giraffe:
+install_giraffe: GIT_SHA1=$(shell git rev-parse HEAD)
+install_giraffe: IMAGE_TAG=$(GIT_SHA1)
+install_giraffe: NAMESPACE=$(OBPS_NAMESPACE_PREFIX)-$(ENVIRONMENT)
```

Comment:
Since we're deploying to the CIF tools namespace, this line can change to 
`install_giraffe: NAMESPACE=$(CIF_NAMESPACE_PREFIX)-tools`